package-lock.json

```diff
@@ -6441,4 +6441,4 @@
       }
     }
   }
-}
+}
```

Comment:
There is still this tiny tiny change, can you undo it @DevXiaolan? 🙏

Reply:
Fix it